ddtrace/internal/serverless/mini_agent.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os
+from platform import python_version_tuple
+import stat
+from subprocess import Popen
+
+from . import in_gcp_function
+from ..logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def maybe_start_serverless_mini_agent():
+    if not in_gcp_function():
+        return
+
+    rust_binary_path = os.getenv("DD_MINI_AGENT_PATH")
+    if rust_binary_path is None:
+        (major, minor, _) = python_version_tuple()
+        rust_binary_path = (
+            "/layers/google.python.pip/pip/lib/python"
+            + major
+            + "."
+            + minor
+            + "/site-packages/datadog-serverless-agent-linux-amd64/datadog-serverless-trace-mini-agent"
```

Comment:
Can we avoid hardcoding the default rust binary path? 

Is linux amd64 the only supported architecture for gcp lambas? Should we disable this feature if another platform is being used?